### PR TITLE
Create POST route/endpoint to save a question to database  

### LINKED ISSUE #76: Create  a route endpoint to create a question to the database

### LINKED ISSUE #78: Create a route endpoints for retrieving all questions

### PR DESCRIPTION
**Pull request by abdulkadiret**

### CODE REVIEW

File: routes/quizzes.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    res.status(500).json({ message: error.message });
+  }
+});
+
+//Creating a question
+router.post('/', async (req, res) => {
+  const quiz = new Quiz({
+    name: req.body.name,
+    question: req.body.question,
+  });
```
Comment (bonboh) [inline routes/quizzes.js]:
```suggestion
  const { name, question } = req.body;
  const quiz = new Quiz({ name, question });
```
What do you think about writing it this way?

**Review by bonboh — CHANGES_REQUESTED** [review]
Good job, just a minor suggestion. Can you also update to the latest master?

File: routes/quizzes.js (hunk 1 above)
Comment (abdulkadiret) [reply routes/quizzes.js]: I like it, it's elegant! :) 
Done!

**Review by bonboh — CHANGES_REQUESTED** [review]
Looks good, just needs to be updated again with the latest master since I just merged #79

**Comment by abdulkadiret** [comment]
I updated with the latest master :)

**Review by bonboh — APPROVED** [review]

**Pull request merged**